src/main/g8/default.properties

```diff
@@ -2,3 +2,4 @@ name=My Akka HTTP Project
 akka_http_version=10.1.1
 akka_version=2.5.12
 organization=com.example
+package=$organization$
```

Comment:
Although it's not a big deal, I am thinking of replacing the above two properties to:
```INI
organization=com.lightbend
package=$organization$.akka.http
```